src/main/java/frc/robot/util/ProntoSwerveModule.java

```diff
@@ -16,6 +20,9 @@
 import edu.wpi.first.math.kinematics.SwerveModuleState;
 import frc.robot.Constants;
 
+/**
+ * Swerve module specific functions and support that don't belong in the subsystem class 
```

Comment:
this is more of an abstraction so that we can have multiple instances of swerve modules that all have the same methods but different states